packages/theme/src/lib/legacy-theme.ts

```diff
@@ -1451,12 +1451,19 @@ export const legacyTheme = {
           fontFamily: tokens.typographyBody1FontFamily,
           letterSpacing: tokens.typographyBody1LetterSpacing,
           color: tokens.colorTextPrimary,
-          padding: '11px 8px',
-          '&:hover': {
-            backgroundColor: tokens.colorBackgroundBorder,
+          padding: '.75em 1em',
+          borderTopLeftRadius: '.25rem',
+          borderTopRightRadius: '.25rem',
+          border: '1px solid',
+          borderColor: 'transparent',
+          marginBottom: '-1px',
+          '&:hover, &:focus, &:focus-visible': {
+            borderColor: '#e9ecef #e9ecef #dee2e6',
```

Comment:
needs higher contrast for focus. We could go with old reliable, Mr. dark blue outline https://availity.github.io/availity-uikit/v4/components/#Nav-Pills

Reply:
fixed